server/routerlicious/packages/services-utils/src/morganLoggerMiddleware.ts

```diff
@@ -120,6 +120,7 @@ export function jsonMorganLoggerMiddleware(
 				[BaseTelemetryProperties.correlationId]: getCorrelationIdWithHttpFallback(req, res),
 				[CommonProperties.serviceName]: serviceName,
 				[CommonProperties.telemetryGroupName]: "http_requests",
+				[HttpProperties.retryCount]: Number.parseInt(req.query.retry as string ?? "0", 10),
```

Comment:
You could do `Number.parseInt(typeof req.query.retry === "string" ? req.query.retry : 0, 10)`